File: tournaments/reflectString/reflectString.cs
string reflectString(string inputString) {
    var result = new StringBuilder();
    
    for (int i = 0; inputString.Length > i; ++i) {
        result.Append((char)('z' - inputString[i] + 'a'));
    }
    
    return result.ToString();
}

File: tournaments/reflectString/reflectString.cpp
std::string reflectString(std::string inputString) {

  std::string result = "";
  for (int i = 0; i < inputString.size(); i++) {
    int order = inputString[i] - 'a',
        reflectedCharCode = int('a') + 25 - order;
    result += char(reflectedCharCode);
  }

  return result;
}
